guides/common/assembly_using-report-templates.adoc

```diff
@@ -15,7 +15,4 @@ include::modules/proc_importing-report-templates-using-the-api.adoc[leveloffset=
 include::modules/proc_generating-a-list-of-installed-packages.adoc[leveloffset=+1]
 
 ifdef::satellite[]
-include::modules/proc_creating-a-report-template-to-monitor-entitlements.adoc[leveloffset=+1]
-endif::[]
```

Comment:
@bangelic Could this be the reason why the build started failing? Can you try re-adding the endif?